### ISSUE TITLE
Generated Alt Text too long

### ISSUE THREAD

**Issue by paeddym** [DMKEBUSINESSGMBH/typo3-mkcontentai#10]
Sometimes when generating and then approving an Alt Text, the backend module throws a 503 error as the Alt Text is too long to store in the column. Would it be possible to set a maximum character limit for the AltText API to generate in the API call to it?

**Comment by hannesbochmann** [comment]
Hi, thanks for your bug report. Can you provide the error/exception message? Which column in which table is affected? In general it should be possible to limit the maximum number of chars in the returned alt text. But maybe it's a better solution to increase the length of the column in the database.

**Comment by paeddym** [comment]
Hi, here is a screenshot of the error shown: 

![image](https://github.com/user-attachments/assets/7f83ed20-331d-419d-a325-51511c802e8e)

So a possible fix would be either to extend the 'alternative' column's length, if TYPO3 allows for it, set a limit in the request or (less elegantly) trim the received Alt Text down to the allowed limit

**Comment by hannesbochmann** [comment]
Thank you for the information. I will take a look at it but I can't tell you when that's gonna be.

**Comment by paeddym** [comment]
No problem, take your time

**Comment by hannesbochmann** [comment]
The error should be fixed with latest release.
